docs/course_materials/compound_data_types.html

```diff
@@ -189,32 +244,171 @@ <h1><b>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        my_tuple = (1, 2, 3)
+                        a, b, c = my_tuple
+                        print(a) # 1
+                        print(b) # 2
+                        print(c) # 3
                     </code></pre>
                 </section>
                 <section data-background-size="cover" data-background-image="media/exr_background.jpeg" data-markdown="exercises/04_Compound_data_types/class_exercise_2.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
                 </section>
             </section>
+            <!-- Slicing -->
+            <section>
+                <section>
+                    <h1>
+                        <b><span>Slicing</span></b>
+                    </h1>
+                </section>
+                <section>
+                    <h3>
+                        <b>Overview</b>
+                    </h3>
+                    <h4>
+                        Slicing is a feature that extracts sub-sets out of lists, tuples and arrays
+                    </h4>
+                </section>
+                <section>
+                    <h3>
+                        <b>Syntax</b>
+                    </h3>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                            my_list[start:end:step]
+                    </code></pre>
+                    <ul>
+                        <li>start - The index from which the sub-list starts (inclusive)</li>
+                        <li>end - The index at which the sub-list ends (exclusive)</li>
+                        <li>step - How many elements to skip</li>
+                    </ul>
+                </section>
+                <section>
+                    <h3>
+                        <b>Examples</b>
+                    </h3>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        my_list = ['a', 'b', 'c', 'd', 'e']
+                        my_list[1:3] # ['b', 'c']
+                        my_list[1:] # ['b', 'c', 'd', 'e']
+                        my_list[:3] # ['a', 'b', 'c']
+                        my_list[:] # ['a', 'b', 'c', 'd', 'e']
+                        my_list[::2] # ['a', 'c', 'e']
+                    </code></pre>
+                </section>
+                <section>
+                    <h4>Important:</h4>
+                    <ul>
+                        <li>The <b>start</b> element is inclusive, which means that it will be included in the sub-list</li>
+                        <li>The <b>end</b> element is exclusive, which means that it won't be included in the sub-list</li>
+                        <li>The <b>step</b> element equals to 1 by default and doesn't need to be written always</li>
+                    </ul>
+                    <h4>Example:</h4>
```

Comment:
i think its better to use <ul> and not h4

Reply:
I don't see your suggestion =\

Reply:
< li >